almanach/adapters/database_adapter.py

```diff
@@ -91,6 +91,11 @@ def list_entities(self, project_id, start, end, entity_type=None):
         return [build_entity_from_dict(entity) for entity in entities]
 
     @database
+    def get_all_entities_by_id(self, entity_id):
+        entities = self.db.entity.find({"entity_id": entity_id}, {"_id": 0})
```

Comment:
If the instance has been resize multiple times, we will return all streams of the instance. It is the requested behaviour or should we return the last occurrence?